src/libmscore/draw/painter.h

```diff
@@ -158,23 +154,36 @@ class Painter
     static IPaintProviderPtr extended;
 
 private:
+
+    struct State {
+        QRect window;
+        QRect viewport;
+        bool isVxF = false;
```

Comment:
Can you explain please what this flag does?

Reply:
This means that `viewTransform` is installed

Reply:
Then it would be great to rename this flag to "isViewTransformInstalled"

Reply:
I took this solution from the Qt sources.
I had an idea not to stray too far from them.